[2023-01-21T06:05:26.793+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: crypto_coins_etl.create_coins_table manual__2023-01-21T06:05:23.032551+00:00 [queued]>
[2023-01-21T06:05:26.806+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: crypto_coins_etl.create_coins_table manual__2023-01-21T06:05:23.032551+00:00 [queued]>
[2023-01-21T06:05:26.807+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-21T06:05:26.808+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-01-21T06:05:26.808+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-21T06:05:26.821+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_coins_table> on 2023-01-21 06:05:23.032551+00:00
[2023-01-21T06:05:26.829+0000] {standard_task_runner.py:55} INFO - Started process 2915 to run task
[2023-01-21T06:05:26.832+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'crypto_coins_etl', 'create_coins_table', 'manual__2023-01-21T06:05:23.032551+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/crypto_coins_etl.py', '--cfg-path', '/tmp/tmpw2yae5e2']
[2023-01-21T06:05:26.836+0000] {standard_task_runner.py:83} INFO - Job 63: Subtask create_coins_table
[2023-01-21T06:05:26.919+0000] {task_command.py:388} INFO - Running <TaskInstance: crypto_coins_etl.create_coins_table manual__2023-01-21T06:05:23.032551+00:00 [running]> on host ac5a85f5e1cc
[2023-01-21T06:05:26.988+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=crypto_coins_etl
AIRFLOW_CTX_TASK_ID=create_coins_table
AIRFLOW_CTX_EXECUTION_DATE=2023-01-21T06:05:23.032551+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-21T06:05:23.032551+00:00
[2023-01-21T06:05:26.992+0000] {sql.py:254} INFO - Executing:  CREATE TABLE IF NOT EXISTS coin(             id INTENGER NOT NULL,             name TEXT NOT NULL,             symbol TEXT ,             current_price FLOAT NOT NULL,             high_24h FLOAT NOT NULL             high_24h FLOAT NOT NULL         );
[2023-01-21T06:05:27.001+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-01-21T06:05:27.255+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-01-21T06:05:27.268+0000] {sql.py:375} INFO - Running statement:  CREATE TABLE IF NOT EXISTS coin(             id INTENGER NOT NULL,             name TEXT NOT NULL,             symbol TEXT ,             current_price FLOAT NOT NULL,             high_24h FLOAT NOT NULL             high_24h FLOAT NOT NULL         );, parameters: None
[2023-01-21T06:05:27.272+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 260, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "high_24h"
LINE 1: ...,             high_24h FLOAT NOT NULL             high_24h F...
                                                             ^

[2023-01-21T06:05:27.287+0000] {taskinstance.py:1318} INFO - Marking task as FAILED. dag_id=crypto_coins_etl, task_id=create_coins_table, execution_date=20230121T060523, start_date=20230121T060526, end_date=20230121T060527
[2023-01-21T06:05:27.302+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 63 for task create_coins_table (syntax error at or near "high_24h"
LINE 1: ...,             high_24h FLOAT NOT NULL             high_24h F...
                                                             ^
; 2915)
[2023-01-21T06:05:27.327+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-01-21T06:05:27.353+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
